pb-remote.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/python
+#!/usr/bin/python
```

Comment:
`#!/usr/bin/env python3` for portability :)

(Should also be a seperate commit)